modules/friendlytag.js

```diff
@@ -28,7 +28,7 @@ Twinkle.tag = function friendlytag() {
 		Twinkle.addPortletLink( Twinkle.tag.callback, "Tag", "friendly-tag", "Add maintenance tags to file" );
 	}
 	// article/draft article tagging
-	else if( ( mw.config.get('wgNamespaceNumber') === 0 || mw.config.get('wgNamespaceNumber') === 118 || /^Wikipedia( talk)?:Articles for creation\//.exec(Morebits.pageNameNorm) ) && mw.config.get('wgCurRevisionId') ) {
+	else if( [0,118].includes(mw.config.get('wgNamespaceNumber')) && mw.config.get('wgCurRevisionId') ) {
```

Comment:
As much as I'd like to ignore IE, we should probably support IE 11 as MediaWiki does. Hence `includes` isn't an option... yet!

Reply:
Jeez! Microsoft would be doing every web developer on the planet a great favour if they just kill IE, and kill it with fire ... 
@MusikAnimal How about creating a shim in morebits.js to support this function? 
`if (!Array.prototype.includes) { Array.prototype.includes = function(el) { return this.indexOf(el) !== -1; } }`
I find `indexOf( ) !== -1` to be quite confusing, unintuitive and unreadable. Even after some months of JS, I still confuse `!== -1` with `=== -1` and vice versa.

Reply:
Fixed for now.